eslint.config.mjs

```diff
@@ -0,0 +1,13 @@
+import { defineConfig } from "eslint/config";
+
+import { default as lostificationConfig } from "eslint-config-lostfictions";
+
+export default defineConfig([
+  lostificationConfig,
+  {
+    rules: {
+      "@typescript-eslint/no-unsafe-argument": "off",
+      "@typescript-eslint/no-unsafe-assignment": "off",
+    },
+  },
+]);
```

Comment:
i've just released `eslint-config-lostfictions@7.0.0` which disables them by default, so this should be fine again if you bump the package! Sorry for not explaining this more clearly.

```suggestion
export { default } from "eslint-config-lostfictions";
```